interface/src/app/analyze/analysis/analysis.js

```diff
@@ -38,12 +48,7 @@ function AnalysisCtrl($scope, $log, $q, $state, SampleBin) {
       {'typename': 'medium'},
       {'typename': 'temperature'},
       {'typename': 'treatment'}
-    ],
-    updateMlModel: function(value) {
-      if (value !== null && value.id !== null) {
-        SampleBin.getActivityForSampleList(value.id);
-      }
-    }
+    ]
```

Comment:
I got a lot of errors like this on the console:
```
TypeError: Cannot read property 'showing' of undefined
    at Object.item_style (adage-0.7.0b.js:41)
    at fn (eval at compile (adage-0.7.0b.js:30), <anonymous>:4:345)
    at o.$digest (adage-0.7.0b.js:28)
    at o.$apply (adage-0.7.0b.js:28)
    at i (adage-0.7.0b.js:27)
    at u (adage-0.7.0b.js:27)
    at XMLHttpRequest.x.onload (adage-0.7.0b.js:27)
```
Does this ring a bell for you?